typings/externs.d.ts

```diff
@@ -183,12 +183,13 @@ declare global {
         };
         frame?: string;
         name?: string;
+        labels?: string;
       };
       pid: number;
       tid: number;
       ts: number;
       dur: number;
-      ph: 'B'|'b'|'D'|'E'|'e'|'F'|'I'|'M'|'N'|'n'|'O'|'R'|'S'|'T'|'X';
+      ph: string; // one of 'B'|'b'|'D'|'E'|'e'|'F'|'I'|'M'|'N'|'n'|'O'|'R'|'S'|'T'|'X';
```

Comment:
any reason? Seems to only use `X` and `M` in this PR, which are already in here

Reply:
fixed